content/600-about/01-about-the-docs.mdx

```diff
@@ -6,9 +6,19 @@ metaDescription: 'This page gives a meta-overview about different topics that ar
 
 ## Overview
 
+`User` and `Post` are the canonical models that are being used throughout the Prisma docs. This page gives some context on why these have been selected and how to interpret them.
+
+
 ## The `User` and `Post` data model
 
-`User` and `Post` are the canonical models that are being used throughout the Prisma docs. This page gives some context on why these have been selected and how to interpret them.
+The `User` and `Post` models have been selected for the following reasons:
+
+- These two models don't require domain specific knowledge.
+- They're also commonly used as an example in the ORM space, making them familiar for users coming from other tools.
+- Having consistent models makes it easier for the reader when learning about different concepts, since there wil be less context switching.
```

Comment:
```suggestion
- Having consistent models makes it easier for the reader when learning about different concepts, since there will be less context switching.
```